mllib/src/test/java/org/apache/spark/mllib/stat/JavaStatisticsSuite.java

```diff
@@ -53,4 +54,12 @@ public void testCorr() {
     // Check default method
     assertEquals(corr1, corr2);
   }
+
+  @Test
+  public void kolmogorovSmirnovTest() {
```

Comment:
If it's not too much trouble, could you also add `chiSqTest` in `JavaStatisticsSuite`?